WebHostLib/static/assets/tutorial/ror2/setup_en.md

```diff
@@ -2,16 +2,12 @@
 
 ## Install using r2modman
 ### Install r2modman
-Head on over to the r2modman page on Thunderstore and follow the installation instructions.
-
-https://thunderstore.io/package/ebkr/r2modman/
+Head on over to the [r2modman page on Thunderstore](https://thunderstore.io/package/ebkr/r2modman/) and follow the installation instructions.
```

Comment:
Links were included separately for accessibility concerns. I'd rather leave them since in-line hyperlinks may be easy to miss for certain users.

Reply:
Put them back as being separate but still as hyperlinks now.